lib/mayaUsd/utils/utilFileSystem.h

```diff
@@ -165,6 +165,11 @@ void updatePostponedRelativePaths(
     const PXR_NS::SdfLayerHandle& layer,
     const std::string&            layerFileName);
 
+/*! \brief Turns the file paths marked through the call 'markPathAsPostponedRelative' to relative.
+ */
+MAYAUSD_CORE_PUBLIC
+void updatePostponedRelativePaths(const PXR_NS::SdfLayerHandle& layer);
```

Comment:
I'm not sure about this but would it be better if we give the layerFileName a default value so we can just keep one method?

Reply:
The default value that would get passed is taken from the layer handle, and we have to account for the possibility that the handle is null. It is simpler to have an overload.